### PR TITLE
btest adding collateral

### PR DESCRIPTION
**Pull request by turadg**
_no tickets_

## Description

Test with production bootstrap of adding a new collateral and running liquidations with it.

### Testing Considerations

Do we want this in RC3? It adds test coverage but it also lightly changes some proposal code and error handling in vaults.

### CODE REVIEW

File: packages/inter-protocol/src/proposals/price-feed-proposal.js [hunk 1]
```diff
@@ -242,6 +256,7 @@ export const getManifestForPriceFeed = async (
     },
   },
   installations: {
+    // ??? will every eval of price-feed-proposal will install priceAggregator ?
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/src/proposals/price-feed-proposal.js]: syntax: "will" appears twice here. One of them is extraneous.

File: packages/vats/test/bootstrapTests/test-liquidation-1.js [hunk 2]
```diff
@@ -129,247 +128,337 @@ test.after.always(t => {
...
+        t.is(nextDescendingStepTime.absValue, nextStartTime.absValue);
+      }
+
+      // console.log(collateralBrandKey, 'step 10 of 10');
+      // readLatest('published.auction.schedule');
+      // await advanceTimeBy(3, 'minutes');
+
+      // console.log(collateralBrandKey, 'step 11 of 10');
+      // readLatest('published.auction.schedule');
+      // await advanceTimeBy(3, 'minutes');
```
Comment (Chris-Hibbert) [inline packages/vats/test/bootstrapTests/test-liquidation-1.js]: drop these?

**Review by Chris-Hibbert — APPROVED** [review]
some nits.  LGTM.   I think this is fine to be merged to master.

File: packages/inter-protocol/src/proposals/price-feed-proposal.js (hunk 1 above)
Comment (turadg) [reply packages/inter-protocol/src/proposals/price-feed-proposal.js]: done

File: packages/vats/test/bootstrapTests/test-liquidation-1.js (hunk 2 above)
Comment (turadg) [reply packages/vats/test/bootstrapTests/test-liquidation-1.js]: done

**Pull request merged**